source/scpp/src/crypto/SecretKey.cpp

```diff
@@ -91,7 +93,7 @@ SecretKey::sign(ByteSlice const& bin) const
 {
     assert(mKeyType == PUBLIC_KEY_TYPE_ED25519);
 
-    Signature out(crypto_sign_BYTES, 0);
+    Signature out;
```

Comment:
How about using `Signature out(0);`. Does it work?

Reply:
I will try it.

Reply:
I tried this, and this is what I got.
```
SecretKey.cpp:96:15: error: no matching constructor for initialization of 'stellar::Signature' (aka 'opaque_array<64>')
    Signature out(0);
```